lib/open_mic.rb

```diff
@@ -0,0 +1,31 @@
+class OpenMic
+  attr_reader :details, :location, :date, :performers, :known_jokes, :repeated_jokes
+
+  def initialize(details)
+    @details = details
+    @location = details[:location]
+    @date = details[:date]
+    @performers = []
+    @known_jokes = []
+    @repeated_jokes = 0
+  end
+
+  def welcome(performer)
+    @performers << performer
+  end
+
+  def repeated_jokes?
+    @performers.each do |performer|
+      performer.jokes.each do |joke|
+        if @known_jokes.include?(joke) == false
```

Comment:
Because `include?` evaluates to a boolean, we don't need to use `== false`. This could be re-written as `if !@known_jokes.include?(joke)`.